components/Left/Texture/Selector.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-row class="pa-0 align-self-center">
+    <v-col cols="1" class="pa-0 align-self-center">
+      <v-container v-if="object_tree[object_tree_index].textures.length > 1" class="pa-0">
```

Comment:
I do not like these container... It's a component for high level in the DOM tree. At least I think..